tests/LineScanCameraTests.cpp

```diff
@@ -46,87 +46,62 @@ TEST_F(ConstVelocityLineScanSensorModel, Inversion) {
   EXPECT_DOUBLE_EQ(imagePt.samp, imageReprojPt.samp);
 }
 
-TEST_F(ConstVelocityLineScanSensorModel, OffBody1) {
+TEST_F(ConstVelocityLineScanSensorModel, OffBody) {
    csm::ImageCoord imagePt(4.5, 4.0);
    csm::EcefCoord groundPt = sensorModel->imageToGround(imagePt, 0.0);
    EXPECT_NEAR(groundPt.x, 0.063995905, 1e-8);
    EXPECT_NEAR(groundPt.y, -7.999488033, 1e-8);
    EXPECT_NEAR(groundPt.z, 8, 1e-8);
 }
-TEST_F(ConstVelocityLineScanSensorModel, OffBody2) {
-   csm::ImageCoord imagePt(4.5, 12.0);
-   csm::EcefCoord groundPt = sensorModel->imageToGround(imagePt, 0.0);
-   // EXPECT_NEAR(groundPt.x, 0.44979759, 1e-8);
-   // EXPECT_NEAR(groundPt.y, 14.99325304, 1e-8);
-   // EXPECT_NEAR(groundPt.z, -14.99325304, 1e-8);
-}
 
-TEST_F(ConstVelocityLineScanSensorModel, OffBody3) {
-   csm::ImageCoord imagePt(12.5, 4.0);
-   csm::EcefCoord groundPt = sensorModel->imageToGround(imagePt, 0.0);
-   // EXPECT_NEAR(groundPt.x, 0.44979759, 1e-8);
-   // EXPECT_NEAR(groundPt.y, 14.99325304, 1e-8);
-   // EXPECT_NEAR(groundPt.z, 14.99325304, 1e-8);
+TEST_F(ConstVelocityLineScanSensorModel, ProximateImageLocus) {
+   csm::ImageCoord imagePt(8.5, 8.0);
+   csm::EcefCoord groundPt(5, 5, 5);
+   csm::EcefLocus locus = sensorModel->imageToProximateImagingLocus(imagePt, groundPt);
+   EXPECT_DOUBLE_EQ(locus.direction.x, -1.0);
+   EXPECT_DOUBLE_EQ(locus.direction.y,  0.0);
+   EXPECT_DOUBLE_EQ(locus.direction.z,  0.0);
+   EXPECT_DOUBLE_EQ(locus.point.x,      5.0);
+   EXPECT_DOUBLE_EQ(locus.point.y,      0.0);
+   EXPECT_DOUBLE_EQ(locus.point.z,      0.0);
```

Comment:
This point is the point on the imaging ray closest to the input ground point (AKA the projection of the ground point on the image ray).

Reply:
The direction is just the direction of the image ray